### ISSUE TITLE
Chat goes oversize after opening a media in landscape mode

### ISSUE THREAD

**Issue by tzs1** [signalapp/Signal-iOS#4043]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
After opening a media in landscape mode, the chat goes extra wide.

### Steps to reproduce
- open a chat
- click on a media
- turn the phone to go on landscape mode viewer
- close the media viewer

**Actual result:** chat width is now x2 larger than before, you can see sent messages without swiping to the right.

**Expected result:** chat width remains the same.

### Screenshots

![image](https://user-images.githubusercontent.com/31325316/48602832-c1b2d180-e974-11e8-88cf-1ca88049be27.png)
![image](https://user-images.githubusercontent.com/31325316/48602838-c6778580-e974-11e8-9aff-400f926386cd.png)

### Device info

**Device**: iPhone 8 Plus

**iOS version**: 12.1

**Signal version:** 2.32.0.11

**Comment by michaelkirk-signal** [comment]
Thanks for being a beta tester.

Do you see this every time you perform the above steps? 

This isn't something I can readily reproduce. Any other details you can provide to help get to the bottom of this?

**Comment by ethylmorphine** [comment]
I failed in my attempts to reproduce this bug.

**Comment by u32i64** [comment]
100% reproducible for me on `iPhone 6 Plus`, Signal `2.32.0.13`

**Comment by tzs1** [comment]
@michaelkirk-signal: yes, I always get the same bug. I can’t find a specific casuistry.

**Comment by michaelkirk-signal** [comment]
I still cannot reproduce this. Are there any more details you can provide @tzs1 

@u32i64 - what steps exacrtly are you taking to reproduce this?

**Comment by u32i64** [comment]
Signal `2.32.0.22` fixed this for me.

**Comment by tzs1** [comment]
It works! Thank you.
